roles/sbs/tasks/main.yml

```diff
@@ -34,6 +34,13 @@
     dest: /etc/apache2/sites-enabled/sbs.conf
   notify: restart Apache
 
+- name: Comment MinProtocol in openssl.conf on dockers
```

Comment:
Is dit echt nodig?  Als het even kan zou ik toch wel by default oude SSL meuk willen uitzetten

Reply:
TLSv1.2 wordt niet ondersteund, verder staat het je vrij om alle SSL varianten uit te zetten, dit was de snelste route naar succes.

Reply:
Ja, maar die openssl conf raakt _alle_ applicaties die openssl gebruiken.  Ik zie dat mariadb 10.4 wle normale TLS support heeft.  Extra reden wellicht om toch over te stappen op een standaard mariadb docker image voor de vm?

Reply:
Uiteindelijk wil ik in de VM eigenlijk zo dicht mogelijk bij de AWS-setup blijven; dat zou dus eigenlijk betekenen dat we een MySQL 5.7 moeten gaan gebruiken (en die doet wel gewoon TLS1.2).  Maar dat is dan misschien even ene ander projectje en moeten we hier idd even een uitzondering maken.